microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -313,3 +346,23 @@ def _gpu_provision_check(min_pci_count: int, node: Node, log: Logger) -> None:
     assert_that(len(curr_gpu)).described_as(
         "GPU PCI device count should be same after stop-start"
     ).is_equal_to(len(init_gpu))
+
+
+def __remove_sources_added_by_extension(
+    node: Node, sources_before: List[str], sources_after: List[str]
+) -> None:
+    rm_sources = [source for source in sources_after if source not in sources_before]
+    for source in rm_sources:
+        node.execute(f"rm /etc/apt/sources.list.d/{source}", sudo=True)
+
+
+def __install_driver_using_sdk(node: Node, log: Logger, log_path: Path) -> None:
+    gpu_feature = node.features[Gpu]
+    gpu_feature.install_compute_sdk()
+    log.debug(
+        f"{gpu_feature.get_supported_driver()} sdk installed. "
+        "Will reboot to load driver."
+    )
+
+    reboot_tool = node.tools[Reboot]
```

Comment:
I recall it also works without reboot, have you met it doesn't work without reboot?

Reply:
Yeah, the test is passing without reboot, shall I remove "reboot_and_check_panic"

Reply:
As per our discussion, we will keep the reboot, since we want to check "reboot_and_check_panic" scenario.